tests/v2/test_1374-to-rdataframe.py

```diff
@@ -197,10 +197,13 @@ class RWonkyDS final : public ROOT::RDF::RDataSource {{
     )
     assert done is True
 
-    rdf = ROOT.awkward.MakeWonkyDataFrame(
+    data_frame = ROOT.awkward.MakeWonkyDataFrame(
         len(example1), lookup1.arrayptrs.ctypes.data, lookup2.arrayptrs.ctypes.data
     )
```

Comment:
We don't really need formal tests of RWonkyDS; it was just a stepping stone to getting the implementation done.

Reply:
removed